src/api/pool_api.js

```diff
@@ -552,6 +555,13 @@ module.exports = {
                 identity: {
                     $ref: 'common_api#/definitions/access_key'
                 },
+                fs_path: {
+                    type: 'string'
+                },
+                fs_backend: {
+                    type: 'string',
+                    enum: ['CEPH_FS', 'GPFS']
```

Comment:
Lets move this to definitions - we don't want to repeat enums multiple times

Reply:
Also @nimrod-becker - I think we want to add NFSv4 to the list as this will be a common use case - what do you say? I thought that using empty might be good for non-specific cases, but with NFSv4 I think it can get more specific than posix - WDYT?

Reply:
Oh @romayalon I see you already added it to definitions so just need to replace with $ref here and other occurrences of it

Reply:
added NFSv4 and moved the definition to common api and replaced all occurrences to $ref

Reply:
@nimrod-becker Would still be good to get your take on NFSv4 here and if we need other granularity of NFSv4.1 or NFSv3 etc.

Reply:
adding NFSv4 as the default is good, I don't think we'll start actually using the specific features of 4.1/4.2 or having to understand the differences between 3 and 4.
I think we can keep 1 for now.

Reply:
OK But I would still make this an optional field, and when not provided use the lowest common denominator possible of POSIX. I consider NFSv4 as a backend with more capabilities than POSIX.

Reply:
Sure, we can do POSIX as default and NFSv4 as higher